Pyrado/pyrado/exploration/normal_noise.py

```diff
@@ -87,7 +87,7 @@ def __init__(self,
 
         # Initialize parameters
         self.log_std_init = to.log(to.tensor(std_init)) if isinstance(std_init, float) else to.log(std_init)
-        self.std_min = to.tensor(std_min) if isinstance(std_min, float) else std_min
+        self.std_min = to.tensor(float(std_min)) if isinstance(std_min, (float, int)) else std_min
```

Comment:
This will work, but using` to.tensor(std_min, dtype=to.get_default_dtype())` seems cleaner to me. However, if you dont want to change it, I am also fine with this solution.

Reply:
Done